src/webextension/list/manage/components/account-summary.css

```diff
@@ -65,13 +65,13 @@
   /* XXX: This positioning is a hack and assumes that the <AccountSummary/> ends
      up in a particular spot in the UI. It would be nice if we could make this
      more flexible in the future. */
-  top: 50px;
-  right: 2px;
-  min-width: 130px;
-  padding: 8px 12px;
-  border: 1px solid #b1b1b3;
+  top: 60px;
+  right: 16px;
+  min-width: 150px;
+  padding: 8px 0;
+  border: 1px solid #D4D4D5;
   border-radius: 6px;
-  box-shadow: 0 2px 8px rgba(12, 12, 13, 0.1);
+  box-shadow: 0 4px 10px rgba(0, 0, 0, 0.2);
```

Comment:
Where's this color coming from? The Photon spec uses the previous value: https://design.firefox.com/photon/patterns/shadows.html